### PR TITLE
831962-Added SfChip UG Content

### PR DESCRIPTION
**Pull request by ghost**
Description:
Here I have added the OverView, GettingStarted, Customization, Events, DataTemplateSelector, Populating Items and Applying font icon to syncfusion control content in the .NET MAUI SfChips control.

### CODE REVIEW

File: MAUI/Chips/Chips/Chips Types.md [hunk 1]
```diff
@@ -0,0 +1,951 @@
...
+    }
+
+}
+
+{% endhighlight %}
+
+![Action typed in Xamarin.Forms SfChipGroup](images/items/Action.png)
+
+
+## See also
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Chips Types.md:947]: Remove the see also

File: MAUI/Chips/Chips/Chips Types.md [hunk 2]
```diff
@@ -0,0 +1,951 @@
...
+	}
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Filter
+
+Allows users to select more than one chip in a group of chips. The selected chips are indicated by selection indicator in this type. The selection indicator can be customized using the [`SelectionIndicatorColor`] property. Use the [`SelectedItems`] property to get the list of selected chips.
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Chips Types.md:521]: Change into SelectedItem

File: MAUI/Chips/Chips/Chips Types.md [hunk 3]
```diff
@@ -0,0 +1,951 @@
...
+<Entry x:Name="entry"
+			VerticalOptions="Center" HeightRequest="40"
+			FontSize="15" 
+			WidthRequest="110" 
+			Completed="entry_Completed"
+			Margin="10,10,0,0">
+                </Entry>
+            </chip:SfChipGroup.InputView>
+<VisualStateManager.VisualStateGroups>
+    <VisualStateGroup x:Name="CommonStates">
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Chips Types.md:34]: Visual state code not needed for input type

File: MAUI/Chips/Chips/Chips Types.md [hunk 4]
```diff
@@ -0,0 +1,951 @@
...
+
+public MainPage()
+	{
+		InitializeComponent();
+
+        VisualStateGroupList visualStateGroupList = new VisualStateGroupList();
+        VisualState normalState = new VisualState() { Name="Normal"};
+
+        VisualStateGroup commonStateGroup = new VisualStateGroup();
+        if (this.ChipType == SfChipsType.Filter || this.ChipType == SfChipsType.Choice)
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Chips Types.md:269]: mention choice type only

File: MAUI/Chips/Chips/Chips Types.md [hunk 5]
```diff
@@ -0,0 +1,951 @@
+---
+layout: post
+title: Chips Types in .NET MAUI Chips control | Syncfusion
+description: Learn about Chips Types support in Syncfusion Essential Studio .NET MAUI Chips control, its elements and more.
+platform: .NET MAUI
+control: Chips
+documentation: ug
+---
+
+# Chips Types in Xamarin Chips
```
Comment (Choza-rajan) [inline MAUI/Chips/Chips/Chips Types.md:10]: Please see here used Xamarin

File: MAUI/Chips/Chips/Chips Types.md [hunk 6]
```diff
@@ -0,0 +1,951 @@
...
+[`SingleOrNone`] - Unlike [`Single`] it is possible to deselect the selected item and keep all the items in deselected state.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<ContentPage
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:core ="clr-namespace:Syncfusion.Maui.Core;assembly=Syncfusion.Maui.Core"
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Chips Types.md:471]: no need to add all the code

File: MAUI/Chips/Chips/Chips Types.md [hunk 7]
```diff
@@ -0,0 +1,951 @@
...
+			</core:SfChipGroup.Items>
+		</core:SfChipGroup>
+	</ContentPage.Content>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.XForms.Buttons;
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Chips Types.md:492]: Remove the unwanted name space

File: MAUI/Chips/Chips/Chips Types.md [hunk 8]
```diff
@@ -0,0 +1,951 @@
...
+
+public MainPage()
+	{
+		InitializeComponent();
+
+        VisualStateGroupList visualStateGroupList = new VisualStateGroupList();
+        VisualState normalState = new VisualState() { Name="Normal"};
+
+        VisualStateGroup commonStateGroup = new VisualStateGroup();
+        if (this.ChipType == SfChipsType.Filter || this.ChipType == SfChipsType.Choice)
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Chips Types.md:568]: Mention chip type filter only.

File: MAUI/Chips/Chips/DataTemplateSelector.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: DataTemplateSelector in .NET MAUI Chips control | Syncfusion
+description: Learn about DataTemplateSelector support in Syncfusion .NET MAUI Chips control, its elements and more.
+platform: .NET MAUI
+control: Chips
+documentation: ug
+---
+
+# DataTemplateSelector in Xamarin Chips
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/DataTemplateSelector.md:10]: removed the xamarin

File: MAUI/Chips/Chips/Chips Types.md [hunk 10]
```diff
@@ -0,0 +1,951 @@
...
+}
+
+{% endhighlight %}
+
+![Action typed in Xamarin.Forms SfChipGroup](images/items/Action.png)
+
+
+## See also
+
+
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Chips Types.md:949]: Remove unwanted space

File: MAUI/Chips/Chips/Customization.md [hunk 11]
```diff
@@ -0,0 +1,2476 @@
+---
+layout: post
+title: Customization in .NET MAUI Chips control | Syncfusion
+description: Learn about Customization support in Syncfusion Essential Studio .NET MAUI Chips control, its elements and more.
+platform: .NET MAUI
+control: Chips
+documentation: ug
+---
+
+# Customization in Xamarin Chips
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Customization.md:10]: Check and remove xamarin content

File: MAUI/Chips/Chips/Events.md [hunk 12]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Events in .NET MAUI Chips control | Syncfusion
+description: Learn about Events support in Syncfusion Essential Studio .NET MAUI Chips control, its elements and more.
+platform: .NET MAUI
+control: Chips
+documentation: ug
+---
+
+# Events in Xamarin Chips
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Events.md:10]: remove the xamarin.forms content

File: MAUI/Chips/Chips/Getting-Started.md [hunk 13]
```diff
@@ -0,0 +1,511 @@
...
+
+After installing [Essential Studio for .NET MAUI](https://www.syncfusion.com/downloads/maui), you can find all the required assemblies in the installation folders, {Syncfusion Essential Studio Installed location} \Essential Studio\\{Version #}\.NET MAUI\lib.
+
+E.g.: C:\Program Files (x86) \Syncfusion\Essential Studio\19.1.0.54\.NET MAUI\lib
+
+N> Assemblies can be found in unzipped package location(Documents/Syncfusion/{Version #}/.NET MAUI/lib) in Mac.
+
+## Adding .NET MAUI Chips reference
+
+Syncfusion Xamarin components are available in [nuget.org](https://www.nuget.org/). To add Chips to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Core](https://www.nuget.org/packages/Syncfusion.Maui.Core), and then install it.
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Getting-Started.md:24]: remove xamarin content

File: MAUI/Chips/Chips/Getting-Started.md [hunk 14]
```diff
@@ -0,0 +1,511 @@
...
+          
+SfChips chips = new SfChips(); 
+Content = chips;  
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+### Android
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Getting-Started.md:106]: Remove the content

File: MAUI/Chips/Chips/Getting-Started.md [hunk 15]
```diff
@@ -0,0 +1,511 @@
...
+    x:Class="Chips.GettingStarted">
+	<ContentPage.Content>
+		<Grid>
+			<core:SfChipGroup>
+				<core:SfChipGroup.ChipLayout>
+					<FlexLayout 
+						HorizontalOptions="Start" 
+						VerticalOptions="Center" 
+						SelectionIndicatorColor="White"  
+                        Command="{Binding ActionCommand}"
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Getting-Started.md:194]: check and change the ode snippet

File: MAUI/Chips/Chips/Getting-Started.md [hunk 16]
```diff
@@ -0,0 +1,511 @@
...
+				ChipPadding="8,8,0,0" 
+				DisplayMemberPath="Name">
+				<core:SfChipGroup.ChipLayout>
+					<FlexLayout 
+						HorizontalOptions="Start" 
+						VerticalOptions="Center" 
+						SelectionIndicatorColor="White"  
+                        Command="{Binding ActionCommand}"   
+                        DisplayMemberPath="Name" 
+                        FlowDirection="LeftToRight"
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Getting-Started.md:334]: check and add proper code snippet

File: MAUI/Chips/Chips/Populating Items.md [hunk 17]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Populating Items in .NET MAUI Chips control | Syncfusion
+description: Learn about Populating Items support in Syncfusion Essential Studio .NET MAUI Chips control, its elements and more.
+platform: .NET MAUI
+control: Chips
+documentation: ug
+---
+
+# Populating Items in Xamarin Chips
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Populating Items.md:10]: remove the xamarin content

File: MAUI/Chips/Chips/Populating Items.md [hunk 18]
```diff
@@ -0,0 +1,82 @@
...
+	}
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Collection of items to chip group](images/items/chips_items.png)
+
+## See also
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/Populating Items.md:82]: Remove see also

File: MAUI/Chips/Chips/how-to/Applying-fonticon-to-Syncfusion-chip-control.md [hunk 19]
```diff
@@ -0,0 +1,70 @@
...
+
+# Setting the FontIcon to SfChip
+
+SfChip is supported to display the font icon by setting `FontImageSource` to its [ImageSource] property with following below steps.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<dict>
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips/how-to/Applying-fonticon-to-Syncfusion-chip-control.md:18]: Remove the code

File: MAUI/Chips/Chips Types.md [hunk 20]
```diff
@@ -0,0 +1,776 @@
...
+
+The following code illustrates how to get an input type chip.
+
+{% highlight xaml %}
+<chip:SfChipGroup 
+                     Margin="200,100,0,0"  
+                     ItemsSource="{Binding Employees}" 
+                     DisplayMemberPath="Name" 
+                     ChipType="Input">
+<chip:SfChipGroup.InputView>
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips Types.md:24]: Remove the underline under code snippet

File: MAUI/Chips/Chips Types.md [hunk 21]
```diff
@@ -0,0 +1,776 @@
...
+        {
+            return selectedEmployee;
+        }
+        set
+        {
+            selectedEmployee = value;
+            OnPropertyChanged(nameof(SelectedEmployee));
+        }
+     }
+    public Command ActionCommand { get; set; }
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips Types.md:114]: Remove action command selected item related code snippet

File: MAUI/Chips/Chips Types.md [hunk 22]
```diff
@@ -0,0 +1,776 @@
...
+
+        Employees.Add(new Employee
+        {
+            Name = "Anne Joseph",
+
+        });
+        Employees.Add(new Employee
+        {
+            Name = "Andrew Fuller",
+
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips Types.md:153]: Change into single line of code

File: MAUI/Chips/Chips Types.md [hunk 23]
```diff
@@ -0,0 +1,776 @@
...
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Core;
+
+public MainPage()
+	{
+		InitializeComponent();
+
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips Types.md:220]: check the code alignment

File: MAUI/Chips/Chips Types.md [hunk 24]
```diff
@@ -0,0 +1,776 @@
...
+        {
+            return selectedEmployee;
+        }
+        set
+        {
+            selectedEmployee = value;
+            OnPropertyChanged(nameof(SelectedEmployee));
+        }
+     }
+    public Command ActionCommand { get; set; }
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips Types.md:317]: remove the action command

File: MAUI/Chips/Chips Types.md [hunk 25]
```diff
@@ -0,0 +1,776 @@
...
+<ContentPage
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:core ="clr-namespace:Syncfusion.Maui.Core;assembly=Syncfusion.Maui.Core"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:local="clr-namespace:Chips"
+    x:Class="Chips.GettingStarted">
+	<ContentPage.Content>
+		<core:SfChipGroup Type="Choice" ChoiceMode="SingleOrNone">
+			<core:SfChipGroup.Items>
```
Comment (jeya-kasipandi) [inline MAUI/Chips/Chips Types.md:400]: Remove the underline

File: MAUI/Chips/Customization.md [hunk 26]
```diff
@@ -0,0 +1,1863 @@
...
+                ItemsSource="{Binding Employees}"
+                DisplayMemberPath="Name"
+                ChipBackground="White"
+                ChipTextColor="Green"
+                ChipType="Choice">
+            <VisualStateManager.VisualStateGroups>
+                    <VisualStateGroup x:Name="CommonStates">
+                        <VisualState x:Name="Normal">
+                            <VisualState.Setters>
+                                <Setter Property="ChipTrextColor" Value="Black" />
```
Comment (Choza-rajan) [inline MAUI/Chips/Customization.md:1260]: See the property name is wrong

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**